packages/slice-machine/src/tracker.ts

```diff
@@ -0,0 +1,79 @@
+import type { Analytics as ClientAnalytics } from "@segment/analytics-next";
+import { AnalyticsBrowser } from "@segment/analytics-next";
+
+// These events should be sync with the tracking Plan on segment.
+
+type AllSliceMachineEventType = EventType | ContinueOnboardingType;
+
+enum EventType {
+  Review = "SliceMachine Review",
+  OnboardingStart = "SliceMachine Onboarding Start",
+  OnboardingSkip = "SliceMachine Onboarding Skip",
+}
+
+export enum ContinueOnboardingType {
+  OnboardingContinueIntro = "SliceMachine Onboarding Continue Screen Intro",
+  OnboardingContinueScreen1 = "SliceMachine Onboarding Continue Screen 1",
+  OnboardingContinueScreen2 = "SliceMachine Onboarding Continue Screen 2",
+  OnboardingContinueScreen3 = "SliceMachine Onboarding Continue Screen 3",
+}
+
+let _client: ClientAnalytics | null = null;
```

Comment:
I don't think it's a good idea to have a mutable singleton in a module instead of a proper class. It's less testable and more error prone.